libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -208,18 +213,24 @@ void AC_WPNav::wp_and_spline_init_cm(float speed_cms, Vector3f stopping_point_ne
 
 /// set_speed_NE_cms - allows main code to pass target horizontal velocity for wp navigation
 void AC_WPNav::set_speed_NE_cms(float speed_cms)
+{
+    set_speed_NE_ms(speed_cms * 0.01);
+}
+
+/// set_speed_NE_cms - allows main code to pass target horizontal velocity for wp navigation
```

Comment:
we should update this comment to start with "set_speed_NE_ms"

Reply:
// Sets the target horizontal speed in cm/s during waypoint navigation.
// See set_speed_NE_ms() for full details.